README.md

```diff
@@ -6,22 +6,26 @@ This Terraform module is designed to configure Amazon Elastic Kubernetes Service
 
 ## Table of Contents
 
-- [Features](#features)
-- [Usage](#usage)
-- [Enabling k8s API Read-Only Access](#enabling-k8s-api-read-only-access)
-  - [Using `eksctl`](#1-using-eksctl)
-  - [Using Terraform](#2-using-terraform)
-- [AWS Documentation](#aws-documentation)
-- [Finishing Steps](#finishing-steps)
-- [Permissions](#permissions)
-- [Example](#example)
-- [Limitations](#limitations)
-- [Requirements](#requirements)
-- [Providers](#providers)
-- [Inputs](#inputs)
-- [Outputs](#outputs)
-- [Resources](#resources)
-- [Contributing](wip.CONTRIBUTING.md)
+- [Terraform AWS EKS](#terraform-aws-eks)
+  - [Table of Contents](#table-of-contents)
```

Comment:
NIT (Non-blocking): Somehow the table of contents got meta here and included the top header and the table of contents itself. I'm not sure if these are generated too, or perhaps this has been manually removed in the past after being generated. It's not a blocker, just something I noticed.